<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSqlTableModelDemoClass</class>
 <widget class="QMainWindow" name="QSqlTableModelDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSqlTabelModelDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_Sort">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="title">
               <string>排序</string>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>22</y>
                 <width>61</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string>排序字段：</string>
               </property>
              </widget>
              <widget class="QComboBox" name="cbSort">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>20</y>
                 <width>121</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="rbAsc">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>50</y>
                 <width>41</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>升序</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="rbDesc">
               <property name="geometry">
                <rect>
                 <x>160</x>
                 <y>50</y>
                 <width>41</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>降序</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_Filter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="title">
               <string>过滤</string>
              </property>
              <widget class="QRadioButton" name="rbBoy">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>20</y>
                 <width>41</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>男</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="rbGril">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>20</y>
                 <width>41</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>女</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="rbBoyAndGril">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>50</y>
                 <width>91</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>全部显示</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTableView" name="tv"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>工  号</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbEmpNo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>姓  名</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbSex">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>性  别</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>身  高</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dtBirthday">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>出生日期</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtMobile">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cbProv">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>广东</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>山东</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>山西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>河南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>河北</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>湖南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>湖北</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西藏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>云南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>四川</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>贵州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>安徽</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>江苏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>浙江</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宁夏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新疆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>内蒙</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黑龙江</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>辽宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>吉林</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>江西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>山西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>福建</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>陕西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>甘肃</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>青海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>台湾</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>出生省份</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtCity">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>城  市</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbPartment">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>销售部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>技术部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>后勤部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>账务部</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>部  门</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbEducation">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>290</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>大学本科</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大学专科</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>研究生</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>学  历</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbSalary">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>320</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>工  资</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>备  注</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="ptMemo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>350</y>
          <width>211</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labPhoto">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>111</width>
          <height>121</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="sbHeight">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="separator"/>
   <addaction name="actAdd"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actSetImage"/>
   <addaction name="actClearImage"/>
   <addaction name="actAddSalary"/>
   <addaction name="actCutSalary"/>
   <addaction name="separator"/>
   <addaction name="actSave"/>
   <addaction name="actCancel"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/MYSL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/MYSL.ICO</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/POSTITL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/POSTITL.ICO</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/NOTEL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/NOTEL.ICO</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/CNFNOT.ICO</normaloff>:/QSqlTabelModelDemo/Resources/CNFNOT.ICO</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actSetImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/MSN.ICO</normaloff>:/QSqlTabelModelDemo/Resources/MSN.ICO</iconset>
   </property>
   <property name="text">
    <string>设置照片</string>
   </property>
  </action>
  <action name="actClearImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/DISTLSTL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/DISTLSTL.ICO</iconset>
   </property>
   <property name="text">
    <string>清除照片</string>
   </property>
  </action>
  <action name="actAddSalary">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/SIGNL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/SIGNL.ICO</iconset>
   </property>
   <property name="text">
    <string>涨工资</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/FOLDER.ICO</normaloff>:/QSqlTabelModelDemo/Resources/FOLDER.ICO</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/EXITEML.ICO</normaloff>:/QSqlTabelModelDemo/Resources/EXITEML.ICO</iconset>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/OOFL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/OOFL.ICO</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actCutSalary">
   <property name="icon">
    <iconset resource="QSqlTableModelDemo.qrc">
     <normaloff>:/QSqlTabelModelDemo/Resources/TASKDECL.ICO</normaloff>:/QSqlTabelModelDemo/Resources/TASKDECL.ICO</iconset>
   </property>
   <property name="text">
    <string>降工资</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSqlTableModelDemo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>QSqlTableModelDemoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
